client_code/popover.py

```diff
@@ -330,6 +334,9 @@ def init_popover(self, element):
         element.setAttribute("ae-popover", "")
         element.setAttribute("ae-popover-id", self.id)
 
+        if self.background:
+            _S(element).css("background-color", self.background)
```

Comment:
Let's not use jquery when vanilla JavaScript will do just fine. Likewise for other uses of jquery

Reply:
done